src/keboola/component/dao.py

```diff
@@ -983,7 +1315,16 @@ def __init__(self, full_path: str,
                  is_public: bool = False,
                  is_permanent: bool = False,
                  is_encrypted: bool = False,
-                 notify: bool = False):
+                 notify: bool = False,
+                 id: str = None,
```

Comment:
@kudj tady je to stejny pripad jako s TableDefinition. vetsina tech veci je specificka jenom pro input manifest a je to matouci, ze to je v konstruktoru. 

`rows_count` urcite neexistuje pro FileManifest, stejne tak `data_size_bytes` To se jmenuje `size_bytes`